mlflow/models/evaluation/base.py

```diff
@@ -778,6 +798,45 @@ def evaluate(
                              If multiple evaluators are specified, each configuration should be
                              supplied as a nested dictionary whose key is the evaluator name.
 
+    :param custom_metrics: (Optional) A list of custom metric functions. A custom metric
+                              function is required to take in two parameters:
+                              - Union[pandas.Dataframe, pyspark.sql.DataFrame]: The first being a
+                                Pandas or Spark DataFrame containing ``prediction`` and ``target``
+                                column. The ``prediction`` column contains the predictions made by
+                                the model. The ``target`` column contains the corresponding labels
+                                to the predictions made on that row.
+                              - Dict: The second is a dictionary containing the metrics calculated
+                                by the default evaluator. The keys are the names of the metrics
+                                and the values are the scalar values of the metrics. Refer to the
+                                DefaultEvaluator behavior section for what metrics will be returned
+                                based on the type of model (i.e. classifier or regressor).
+
+                              A custom metric function can return in two different formats:
+                              - Dict[AnyStr, Union[int, float, np.number]: a singular dictionary of
+                                custom metrics, where the keys are the names of the metrics, and the
+                                values are the scalar values of the metrics.
```

Comment:
Can we add spaces between the list titles and the elements? The current rendering is probably not quite what we intend:

![Uploading Screen Shot 2022-02-23 at 5.42.59 PM.png…]()

Reply:
Fixed! 
<img width="1303" alt="image" src="https://user-images.githubusercontent.com/9274754/155603681-4881e27a-6a1a-452d-bee7-46d32268bbfb.png">